cli/command/manifest/push_test.go

```diff
@@ -51,6 +53,7 @@ func TestManifestPushErrors(t *testing.T) {
 
 // store a one-image manifest list and puah it
 func TestManifestPush(t *testing.T) {
+	skip.If(t, os.Getuid() != 0, "require root to run")
```

Comment:
I'm able to run this without root on linux. How does this fail for you?

Reply:
`push_test.go:71: assertion failed: error is not nil: open /etc/docker/certs.d/example.com: permission denied`

Reply:
The client used by that test should be mocked by `newFakeRegistryClient`, so I'm not sure why it's looking for a cert. I can't reproduce the issue on linux.  Does it happen on both osx and linux for you?

Reply:
Ok, for me `/etc/docker` is `0600` and the test is ok with it.

Reply:
Hum I'll check tomorrow but that ain't big deal 😛